wof/WaterML_1_1.py

```diff
@@ -402,10 +402,11 @@ def quote_xml(inStr):
         pos = mo.end()
     s3 = s1[pos:]
     s2 += quote_xml_aux(s3)
-    return s2.decode(ExternalEncoding)
```

Comment:
This is what we should avoid, the encode/decode whack-a-mole :smile: 

[The principle is:](https://docs.python.org/2/howto/pyporting.html)

```
Decide which of your APIs take text and which take binary data
Make sure that your code that works with text also works with unicode and code for binary data works with bytes in Python 2 (see the table above for what methods you cannot use for each type)
Mark all binary literals with a b prefix, textual literals with a u prefix
Decode binary data to text as soon as possible, encode text as binary data as late as possible
Open files using io.open() and make sure to specify the b mode when appropriate
Be careful when indexing into binary data
```